src/main/groovy/edu/oregonstate/mist/locations/frontend/resources/LocationResource.groovy

```diff
@@ -26,8 +26,10 @@ import java.util.regex.Pattern
 class LocationResource extends Resource {
     private static final Logger LOGGER = LoggerFactory.getLogger(LocationResource.class)
 
+    private final Integer MAX_PAGE_SIZE = 10000
     public static final ArrayList<String> ALLOWED_CAMPUSES = ["corvallis", "extension"]
-    public static final ArrayList<String> ALLOWED_TYPES = ["building", "dining", "cultural-center"]
+    public static final ArrayList<String> ALLOWED_TYPES = ["building", "dining",
```

Comment:
Should the swagger spec, readme or smartdoc be updated to mention the allowed types or max_page size?